src/Compilers/Core/Portable/Operations/OperationNodes.cs

```diff
@@ -363,4 +363,20 @@ internal sealed partial class StaticLocalInitializationSemaphoreOperation
         {
         }
     }
+
+    internal sealed partial class BlockOperation
+    {
+        public static BlockOperation CreateTemporaryBlock(ImmutableArray<IOperation> statements, SemanticModel semanticModel, SyntaxNode syntax)
```

Comment:
>CreateTemporaryBlock [](start = 37, length = 20)

It would be good to explain what is special about this block that we call it "temporary". Are there restrictions on usage, etc. ?

Reply:
Added a doc comment.

---
In reply to: [526904956](https://github.com/dotnet/roslyn/pull/49453#discussion_r526904956) [](ancestors = 526904956)